src/style.css

```
@@ -1 +1,13 @@
-/* styles go here */
+.fieldset {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  padding-bottom: 0;
+  padding-top: 0;
+}
```

```suggestion
.fieldset {
  margin-bottom: 20px;
}
```
Margin top is redundant. Also, there is no need to reset the default paddings